mocha-client-tests/addon/tests/lib/background-messaging.test.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * Created by hmelenok on 10/30/16.
+ */
+describe('Background', function() {
+    describe('ping', function() {
+        it('should return pong in response', function(done) {
```

Comment:
how do I run this test? It isn't executed with `npm run test` from the root directory. Is there a different way to run it?